src/Testing/CMakeLists.txt

```
@@ -0,0 +1,28 @@
+# sub-scripts for testing of MPAGSCipher library
+
+# testHello executable settings
+add_executable(testHello testHello.cpp)
+add_test(NAME test-hello COMMAND testHello)
+
+# testCatch executable settings
+add_executable(testCatch testCatch.cpp)
+target_include_directories(testCatch PRIVATE $(CMAKE_CURRENT_LIST_DIR))
+add_test(NAME test-catch COMMAND testCatch)
+
+# testTransformChar test settings
+add_executable(testTransformChar testTransformChar.cpp)
+target_include_directories(testTransformChar PRIVATE $(CMAKE_CURRENT_LIST_DIR))
+target_link_libraries(testTransformChar MPAGSCipher)
```

```suggestion
target_link_libraries(testTransformChar Catch MPAGSCipher)
```
and similarly for the other tests